modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/xeno.dm

```diff
@@ -52,3 +74,8 @@
 	xeno.dna.mutant_bodyparts["xenohead"] = list(MUTANT_INDEX_NAME = "Standard", MUTANT_INDEX_COLOR_LIST = list(xeno_color, xeno_color, xeno_color))
 	regenerate_organs(xeno, src, visual_only = TRUE)
 	xeno.update_body(TRUE)
+
+/obj/item/organ/internal/alien/plasmavessel/roundstart
+	stored_plasma = 55
+	max_plasma = 55
+	plasma_rate = 2
```

Comment:
This gives them a *very* fast heal rate when standing on resin floors, healing their brute, burn and oxygen damage at a by 0.2*2.5 = 0.5/s each.
For now, I'd rather not have to consider or worry about xeno-hybrids being able to heal using this,
 so just set the `heal_rate` to `0` for the time being.
```suggestion
	plasma_rate = 2
	heal_rate = 0
```